src/AccessibilityInsights/MainWindowHelpers/Initialize.cs

```diff
@@ -83,6 +85,8 @@ void StartStartMode()
             UpdateTabSelection();
         }
 
+        private void BackToSelectingOnDispatcher(Task<bool> _) => Dispatcher.Invoke(HandleBackToSelectingState);
```

Comment:
>_ [](start = 60, length = 1)

Not sure if this is needed?

Reply:
It's for the throwaway task parameter that gets passed in